### ISSUE TITLE
Upgrade geth-utils go1.16 to go1.18

### ISSUE THREAD

**Issue by Lawliet-Chan** [privacy-scaling-explorations/zkevm-circuits#904]
Shall we upgrade go version of [geth-utils](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/geth-utils/go.mod#L3) to 1.18? Like https://github.com/ethereum/go-ethereum/blob/master/go.mod#L3

**Comment by lispc** [comment]
no.   We are using geth 1.10.18, which uses go 1.16.    https://github.com/ethereum/go-ethereum/blob/de23cf910b814867d5c5d1ad6164835d79069638/go.mod#L3 

If in the future we upgrade geth version, we can recheck go version
